### DISCUSSION TITLE
structuredClone instead of JSON?

### DISCUSSION THREAD

**Discussion by sschneider-ihre-pvs** [vueuse/vueuse#2843] [Q&A]
Would it make sense to use structuredClone and only json.parse as fallback?
https://github.com/vueuse/vueuse/blob/192b4a2a6dee20bcca282601da094083c06ef8f4/packages/core/useCloned/index.ts#L35

**Comment by daiwanxing** [comment]
I think using `structuredClone` may not be suitable due to the following concerns:

1. Compatibility: It is only supported by Chrome 98+.
2. Serializing values such as functions and symbols will result in error and crash, while using JSON.stringify will fail silently and skip these values.
